### DISCUSSION TITLE
The program exits abnormally under certain conditions.

### DISCUSSION THREAD

**Discussion by YamatoRyou** [rustdesk/rustdesk#8461] [General]
### Bug Description

If the system starts RustDesk immediately after entering the desktop and tries to connect to a remote desktop, the program will exit immediately. **However, if you wait a few minutes before starting RustDesk, there will be no problem.**

### How to Reproduce

1. Update to 1.2.6 (install as usual to make it resident on the system);
2. Restart Windows;
3. Start RustDesk as soon as you enter the desktop (to operate as quickly as possible), and then connect to a remote desktop (or transfer files).

### Expected Behavior

Available at any time.

### Operating system(s) on local side and remote side

x64 Windows 11 -> x64 Windows 7

### RustDesk Version(s) on local side and remote side

1.2.6 (for 64 Windows) -> 1.2.6 (for 64 Windows)

### Screenshots

**Since this problem only occurs under certain conditions, it is impossible to record the screen content at the time. If there is a chance, I will add it later.**

**Comment by 21pages** [comment]
Thanks,  reproduced.

**Comment by rustdesk** [comment]
This is known by our new design in 1.2.6. We know it impacts user experience, we may change to close and reopen quickly.

---
We got an idea to avoid reopen.

**Comment by rustdesk** [comment]
Apologize, this has nothing to do with our new design. convert back to issue.  https://github.com/rustdesk/rustdesk/issues/8462

**Comment by rustdesk** [comment]
Please try out https://github.com/rustdesk/rustdesk/releases/tag/nightly
